libevm/ExtVMFace.h

```diff
@@ -193,23 +192,21 @@ struct CreateResult
     h160 address;
 
     CreateResult(evmc_status_code status, owning_bytes_ref&& output, h160 const& address)
-        : status{status}, output{std::move(output)}, address{address}
+      : status{status}, output{std::move(output)}, address{address}
     {}
 };
 
 /**
  * @brief Interface and null implementation of the class for specifying VM externalities.
  */
-class ExtVMFace: public evmc_context
+class ExtVMFace : public evmc::Host
```

Comment:
Well this was already a weird class, mixing pure virtual methods with some implementations and data members, and now it's even weirder having two sets of similar methods.

Would it be possible to split this into two, like maybe having `evmc::Host` implemented separately in terms of `ExtVMFace` (passing it to constructor), and then instantiating it inside `EVMC::exec` ?

Reply:
Probably can be done, but it would be better if the ExtVMFace for legacy VM is implemented as the wrapper for evmc::Host and evmc::Host implementation replaces ExtVM.

Reply:
I can try to do that

Reply:
I can leave it to you. I'm currently busy with adding python bindings to ethash library. This PR is not really required, but reuses some code added to EVMC C++.